src/base/utils/os.cpp

```diff
@@ -289,14 +290,16 @@ bool Utils::OS::applyMarkOfTheWeb(const Path &file, const QString &url)
         }
     }
 
-    if (!::SetFilePointerEx(handle, {0}, nullptr, FILE_BEGIN))
+    LARGE_INTEGER zeroPos;
+    zeroPos.QuadPart = 0;
+    if (!::SetFilePointerEx(handle, zeroPos, nullptr, FILE_BEGIN))
```

Comment:
```
/opt/mxe/gits/qbittorrent-5.1.2/src/base/utils/os.cpp:358:28: warning: missing initializer for member '_LARGE_INTEGER::<unnamed struct>::HighPart' [-Wmissing-field-initializers]
  358 |     if (!::SetFilePointerEx(handle, {0}, nullptr, FILE_BEGIN))
      |          ~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
AFAIK assigning `{0}` to `LARGE_INTEGER` should zero initialize **all** member fields. Or am I wrong? At least our all compilers in our CI don't complain about it.

BTW, what compiler version were you using?

Reply:
> BTW, why not use `LARGE_INTEGER zeroPos {}`?

Good point. Seems to work, and much cleaner.

Reply:
> > BTW, why not use `LARGE_INTEGER zeroPos {}`?
> 
> Good point. Seems to work, and much cleaner.

Or you can explicitly initialize `QuadPart` using designated initializer:
```c++
LARGE_INTEGER zeroPos {.QuadPart = 0};
```